sql/search_derived/search_metric_contribution/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+friendly_name: Search Metric Contribution
+description: >
+  A table shows how much of one metric is contributed by another metric.
```

Comment:
This could use a bit more fleshing out: What specific questions does this solve? What are the fields? What is it derived from?